dataproc/src/test/java/CreateClusterTest.java

```diff
@@ -60,35 +60,32 @@ public static void checkRequirements() {
 
   @Before
   public void setUp() {
-    clusterName = String.format("%s-%s", BASE_CLUSTER_NAME, UUID.randomUUID().toString());
-
     bout = new ByteArrayOutputStream();
     System.setOut(new PrintStream(bout));
+    standardOutOrig = System.out;
```

Comment:
You don't need this, but if you do keep it in it needs to set `standardOutOrig` before you call `System.setOut` - otherwise you are just getting a reference to the value you just set.